tests/framework/installer/ceph_manifests.go

```diff
@@ -1675,6 +1676,52 @@ subjects:
 
 // GetRookCluster returns rook-cluster manifest
 func (m *CephManifestsMaster) GetRookCluster(settings *ClusterSettings) string {
+	if settings.UsePVCs {
+		return `apiVersion: ceph.rook.io/v1
+kind: CephCluster
+metadata:
+name: ` + settings.Namespace + `
+namespace: ` + settings.Namespace + `
+spec:
+cephVersion:
+  image: ` + settings.CephVersion.Image + `
+  allowUnsupported: ` + strconv.FormatBool(settings.CephVersion.AllowUnsupported) + `
+dataDirHostPath: ` + settings.DataDirHostPath + `
+network:
+  hostNetwork: false
+mon:
+  count: ` + strconv.Itoa(settings.Mons) + `
+  allowMultiplePerNode: true
+dashboard:
+  enabled: true
+rbdMirroring:
+  workers: ` + strconv.Itoa(settings.RBDMirrorWorkers) + `
+metadataDevice:
+storage:
+  config:
+    storeType: "` + settings.StoreType + `"
+    databaseSizeMB: "1024"
+    journalSizeMB: "1024"
+  storageClassDeviceSets:
+  - name: set1
+    count: 1
+    resources:
+    volumeClaimTemplates:
+    - metadata:
+        name: data
+      spec:
+        resources:
+          requests:
+            storage: 10Gi
```

Comment:
for the tests let's request something small like 50Mi.

Reply:
We can shrink this to 1Gi?

Reply:
done